stdlib/gc.mli

```diff
@@ -145,17 +145,37 @@ type control =
     mutable allocation_policy : int;
     [@ocaml.deprecated_mutable
          "Use {(Gc.get()) with Gc.allocation_policy = ...}"]
-    (** The policy used for allocating in the heap.  Possible
-        values are 0 to 2.  0 is the original next-fit policy,
-        usually fast, but can result in fragmentation.  1 is the
-        first-fit policy, which can be slower in some cases but
-        can be better for programs with fragmentation problems.
-        2 is the best-fit allocation policy, the best for avoiding
-        fragmentation.
-        Note that changing the allocation policy at run-time forces
+    (** The policy used for allocating in the major heap.
+        Possible values are 0, 1 and 2.
+        - 0 is the next-fit policy, which is usually fast
+          but can result in fragmentation.
+        - 1 is the first-fit policy, which avoids fragmentation
+          but is sensibly slower in some real-world use-cases
+          (it is only useful on programs exhibiting fragmentation issues
+           with next-fit)
+        - 2 is the best-fit policy, which is better than best-fit
+          and first-fit: it is both fast at allocating and excellent
```

Comment:
I suggest a wording along the lines of replacing “... which is better than ...” with “... which empirical results suggests it is better than ...” otherwise it still sounds like this is a theoretical result.

Also, “... (since 4.10)”.

Reply:
Also typo: the second best-fit should be replace by next-fit.

Reply:
Thanks to you both, I improved the text. I also added a small paragraph to reproduce some actual performance numbers, to give an idea of what kind of empirical differences we are talking about.